tests/dragonfly/replication_test.py

```diff
@@ -3589,8 +3589,11 @@ async def test_takeover_bug_wrong_replica_checked_in_logs(df_factory):
 
     # Reconnect replica[1] and immediately takeover from replica[0]
     await clients[1].execute_command(f"REPLICAOF localhost {master.port}")
+
+    await asyncio.sleep(0.5)
```

Comment:
this is not how we do it - it's not robust.

check `wait_available_async` , and
```
async with async_timeout.timeout(3):
        await wait_for_replicas_state
```
and `await check_all_replicas_finished` and @assert_eventually
  
yes, we have lots of utilities to do the same thing but sleep is not the right one.